content/projects/bootcamp-writing-assignment/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: Bootcamp Writing Assignment
+---
+
+### We would like to get to know you better.
+
+In this assignment you are going to create a .md file with a 1000 word essay on yourself, in this essay please can you try cover the below points:
+
+ - Your personal history, where you come from, feel free to include some interesting facts about yourself.
```

Comment:
This is good. I think this can replace parts of out soft skills interview, is that the intention?

It might be easier to mark if we tell them to not put it all  in one unstructured essay. Eg the instruction could be like:
"write a maximmum of 200 words for each of the questions below"

Reply:
This one has multiple intentions:
1. Assess their writing and their grammer etc.
2. Get data for their profiles for when we need it.
3. Assess cultural fit.
4. Help us compress or fact check the results of the culture interview